source/observables.txt

```diff
@@ -27,24 +27,30 @@ An ``Observable`` represents a stream of data emitted by an operation
 over time. To access this data, you can create an ``Observer`` instance
 that subscribes to the ``Observable``. 
 
+.. note::
+
+   The {+driver-short+} is built upon the MongoDB Java Reactive Streams driver.
+   The ``Observable`` class extends the ``Publisher`` class from Java Reactive
+   Streams and includes additional convenience methods to help process results.
+
 How to Process an Observable
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 To run a MongoDB operation and process its data, you must request
-the operation results from an ``Observable``. The ``Observable``
-is a high-level abstraction built on top of a cursor, or a 
+the operation results from an ``Observable``. An ``Observable``
+that returns multiple results is built on top of a cursor, or a 
```

Comment:
It's better, but thinking about it more, it's both incomplete and inaccurate

# Not all Observable instances are built on top of a cursor.  Some commands that return multiple results, like `distinct` don't actually use cursors
# If we are going to say what one type of observable is built on, it feels incomplete not to say what the others are built on.

I wonder if we can make this more abstract, and just say that the Observable interface is used for operations that can return an unbounded number of items (including 0, for some operations, like `FindObservable#first`), while SingleObservable is used for operations that return either 0 or 1 items.  And then give an example of each.

What do you think?

Reply:
Okay, I reworded to make this part more generic. I removed the mention of cursors and discussed the SingleObservable for 0/1 results and Observable for an unbounded number. Let me know what you think.